<ui version="4.0" >
 <class>encrWidget</class>
 <widget class="QWidget" name="encrWidget" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>259</width>
    <height>399</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" >
   <item>
    <widget class="QComboBox" name="enctype" />
   </item>
   <item>
    <layout class="QGridLayout" >
     <item row="0" column="0" >
      <widget class="QLabel" name="enclab_1" >
       <property name="text" >
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1" >
      <widget class="QLineEdit" name="enckey_1" />
     </item>
     <item row="1" column="0" >
      <widget class="QLabel" name="enclab_2" >
       <property name="text" >
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1" >
      <widget class="QLineEdit" name="enckey_2" />
     </item>
     <item row="2" column="0" >
      <widget class="QLabel" name="enclab_3" >
       <property name="text" >
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1" >
      <widget class="QLineEdit" name="enckey_3" />
     </item>
     <item row="3" column="0" >
      <widget class="QLabel" name="enclab_4" >
       <property name="text" >
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1" >
      <widget class="QLineEdit" name="enckey_4" />
     </item>
    </layout>
   </item>
   <item>
    <spacer>
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" >
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
